### ISSUE TITLE
About the opencv imwrite function

### ISSUE THREAD

**Issue by leafjungle** [google/guetzli#232]
I did an experiment with two tools:  guetzli and opencv(version:2.4,  imwrite function). And I found that opencv is much faster(100+ seconds   VS  80ms), while the PSNR and compress ratio is almost the same.

So I am so puzzled with the difference between Guetzli and opencv::imwrite?
Is that PSNR is not suitable? or Butteraugli is more suitable?

**Comment by pornel** [comment]
PSNR is a bad metric that very poorly describes quality. It should not be used for evaluation of lossy codecs. See http://www.ponomarenko.info/tid2013.htm

**Comment by jyrkialakuijala** [comment]
Exactly! In PIK we have designed the compression format in a way that allows us to make psychovisually sound quantization much faster (assuming butteraugli is what we want), and the fast mode of PIK gets already pretty close to butteraugli scores while being roughly as fast as traditional JPEG compression.

**Comment by leafjungle** [comment]
@pornel ,  is there an experiment shows the Butteraugli scores of guetzli and libjpeg?  (based on  the same quality)

**Comment by pornel** [comment]
https://arxiv.org/abs/1703.04421

**Comment by HenkPoley** [comment]
btw, there's a new quality metric that supposedly even more closely mimics human scoring of images (better than Butteraugli). 

ssimulacra: https://cloudinary.com/blog/detecting_the_psychovisual_impact_of_compression_related_artifacts_using_ssimulacra

Github repo: https://github.com/cloudinary/ssimulacra
